src/app/clients/ClientsPage.tsx

```diff
@@ -28,12 +28,14 @@ export interface ClientsTableRow {
     fullName: string;
     familyCategory: string;
     addressPostcode: string | null;
+    phoneNumber: string | null;
 }
 
 const headers: TableHeaders<ClientsTableRow> = [
     ["fullName", "Name"],
     ["familyCategory", "Family"],
     ["addressPostcode", "Postcode"],
+    ["phoneNumber", "Phone Number"],
```

Comment:
lets change this to be consistent with the parcels table: `["phoneNumber", "Phone"],`